### PR TITLE
docs - env var 49

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
Adds auto-generated env var docs.

### CODE REVIEW

File: docs/configuring-metabase/environment-variables.md [hunk 1]
```diff
@@ -37,133 +45,123 @@ docker run -d -p 3000:3000 -e MB_SITE_NAME="Awesome Company" --name metabase met
...
-Default: 10000
-
-Maximum number of rows to return for aggregated queries via the API. Must be less than 1048575. This environment variable also affects how many rows Metabase includes in dashboard subscription attachments.
+- Type: integer
+- Default: `10000`
+- [Exported as](../installation-and-operation/serialization.md): `aggregated-query-row-limit`.
+- [Configuration file name](./config-file.md): `aggregated-query-row-limit`
 
-See also [`MB_UNAGGREGATED_QUERY_ROW_LIMIT`](#mb_unaggregated_query_row_limit).
+Maximum number of rows to return for aggregated queries via the API.
```
Comment (alexyarosh) [inline docs/configuring-metabase/environment-variables.md:66]: Would be nice to have the extra context from the previous version

File: docs/configuring-metabase/environment-variables.md [hunk 2]
```diff
@@ -172,1047 +170,1687 @@ See [fonts](../configuring-metabase/fonts.md).
...
-### `MB_AUDIT_MAX_RETENTION_DAYS`
-
-Only available on Metabase [Pro](https://www.metabase.com/product/pro) and [Enterprise](https://www.metabase.com/product/enterprise) plans.<br>
-Type: integer<br>
-Default: 720 (Metabase keeps all rows)<br>
+- Type: positive-integer
+- Default: `20`
 
-Sets the maximum number of days Metabase preserves rows for the following application database tables:
+Maximum number of rows to render in an alert or subscription image.
```
Comment (alexyarosh) [inline docs/configuring-metabase/environment-variables.md:200]: Again, I think the extra context from the original file is important

File: docs/configuring-metabase/environment-variables.md [hunk 3]
```diff
@@ -172,1047 +170,1687 @@ See [fonts](../configuring-metabase/fonts.md).
...
 
 ### `MB_DB_CONNECTION_TIMEOUT_MS`
 
-Type: integer<br>
-Default: `10000`
+- Type: integer
+- Default: `10000`
 
-Timeout in milliseconds for connecting to databases, both Metabase application database and data connections. In case you're connecting via an SSH tunnel and run into a timeout, you might consider increasing this value as the connections via tunnels have more overhead than connections without.
+Consider metabase.driver/can-connect? / can-connect-with-details? to have failed if they were not able to
```
Comment (alexyarosh) [inline docs/configuring-metabase/environment-variables.md:282]: This description isn't helpful to end user

**Review by alexyarosh — APPROVED** [review]
Approving to unblock, but I think we should do a full audit of descriptions. I foudn some variables that have better descriptions in the original file (I didn't comment on all of them, just the first few)

**Review by camsaul — APPROVED** [review]

**Comment by jeff-bruemmer** [comment]
Closing this in favor of https://github.com/metabase/metabase/pull/43150.

**Pull request closed without merging**